src/ServiceControl.UnitTests/ApprovalFiles/RavenPersistedTypes.Verify.approved.txt

```diff
@@ -1,16 +1,10 @@
 ServiceControl.SagaAudit.SagaListIndex+Result, ServiceControl, Version=4.0.0.0, Culture=neutral, PublicKeyToken=null
-ServiceControl.MessageFailures.FailedMessage, ServiceControl, Version=4.0.0.0, Culture=neutral, PublicKeyToken=null
+ServiceControl.MessageFailures.FailedMessage, ServiceControl.Persistence, Version=4.0.0.0, Culture=neutral, PublicKeyToken=null
 ServiceControl.Recoverability.FailureGroupView, ServiceControl, Version=4.0.0.0, Culture=neutral, PublicKeyToken=null
 ServiceControl.Recoverability.FailureGroupMessageView, ServiceControl, Version=4.0.0.0, Culture=neutral, PublicKeyToken=null
-ServiceControl.MessageFailures.GroupComment, ServiceControl, Version=4.0.0.0, Culture=neutral, PublicKeyToken=null
-ServiceControl.Recoverability.FailedMessageRetry, ServiceControl, Version=4.0.0.0, Culture=neutral, PublicKeyToken=null
-ServiceControl.Recoverability.RetryBatch, ServiceControl, Version=4.0.0.0, Culture=neutral, PublicKeyToken=null
-ServiceControl.Recoverability.RetryBatchGroup, ServiceControl, Version=4.0.0.0, Culture=neutral, PublicKeyToken=null
-ServiceControl.Monitoring.KnownEndpoint, ServiceControl, Version=4.0.0.0, Culture=neutral, PublicKeyToken=null
```

Comment:
These should probably still be being picked up. The test probably should be moved in to the RavenDB persistence tests